docs/variants/novacustom_nuc_box/releases.md

```diff
@@ -71,8 +71,6 @@ using [this key](https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/cu
     + [License](https://chromium.googlesource.com/chromiumos/platform/vboot_reference/+/f1f70f46/LICENSE)
 - [Intel Management Engine version v18.0.5.2028](https://github.com/Dasharo/dasharo-blobs/blob/52647f9c/novacustom/nuc_box/me.bin)
     + [License](https://github.com/Dasharo/dasharo-blobs/blob/main/licenses/pv%20intel%20obl%20software%20license%20agreement%2011.2.2017.pdf)
-- [Intel Firmware Support Package for Meteor Lake-H version 2024/04/30 v4122_12](https://github.com/Dasharo/dasharo-blobs/tree/52647f9c/novacustom/nuc_box/MeteorLakeFspBinPkg)
```

Comment:
But it uses the FSP from laptops, no?
https://github.com/Dasharo/coreboot/blob/dasharo/configs/config.novacustom_nuc_box#L16